app/models/question_submission_history.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class QuestionSubmissionHistory < ApplicationRecord
+  self.primary_keys = :question_id, :user_id
```

Comment:
What is the purpose of a composite primary key here? We can just use a unique index, right ?

Reply:
@tachyons here composite primary key is used to uniquely identify the submission of a user for a particular question. for example an user with `id=1` can have multiple `question_id` against it, so to uniquely identify, composite primary key is used. We can use unique index but then we need to add another column. Whats your opinion on it ?